components/stacks-rpc-client/src/rpc_client.rs

```diff
@@ -67,7 +67,33 @@ pub struct PoxInfo {
 }
 
 impl PoxInfo {
-    pub fn default() -> PoxInfo {
+    pub fn mainnet_default() -> PoxInfo {
+        PoxInfo {
+            contract_id: "SP000000000000000000002Q6VF78.pox-2".into(),
+            pox_activation_threshold_ustx: 0,
+            first_burnchain_block_height: 666050,
+            prepare_phase_block_length: 100,
+            reward_phase_block_length: 2000,
+            reward_slots: 4000,
+            total_liquid_supply_ustx: 1368787887756275,
+            ..Default::default()
+        }
+    }
+
+    pub fn testnet_default() -> PoxInfo {
+        PoxInfo {
+            contract_id: "ST000000000000000000002AMW42H.pox-2".into(),
```

Comment:
Could we update to `pox-3`?

Reply:
Done! I think? 🤔 

I got my info from: http://seed.testnet.hiro.so:20443/v2/pox and http://seed.mainnet.hiro.so:20443/v2/pox.

Can you confirm that what I entered here is correct?